scripts/templates/create_user_plain.erb

```diff
@@ -0,0 +1,14 @@
+Hi,
+
+This is your account on GOV.UK PaaS:
+
+Username: <%= email %><% if defined?(org) && org != "" %>
+Organisation: <%= org %><% end %>
```

Comment:
The original version only output this line if the org was set (it's not set when resetting someone's password for example). We should preserve this behaviour.

Reply:
Sorry, I misread this. I think now that this is no longer in bash, we can put the conditional on a newline. We should be able to remove the extra whitespace with using `<%-` tag.

Reply:
As the templates were separated this if is not needed anymore.